dataactbroker/permissions.py

```diff
@@ -39,7 +39,7 @@ def decorated_function(*args, **kwargs):
                                 break
 
                     finally:
-                        InterfaceHolder.closeOne(userDb)
+                        userDb.close()
```

Comment:
Okay--thanks for the explanation. I'll look at this more when digging more deeply int o the db code.